style.css

```diff
@@ -78,3 +78,381 @@ span {
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.scroll-down {
+  margin-left: 140px;
+  text-align: center;
+  border-width: 0.01em;
+  background: white;
+  border-style: solid;
+  border-color: rgb(211, 210, 210);
+  border-radius: 50%;
+}
+
+.indicator {
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.container3 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item3 {
+  margin-bottom: 30px;
+}
+
+.btn-group {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.languages {
+  flex-grow: 2;
+  height: 40px;
+  color: #3a4a42;
+  border-style: none;
+  border-radius: 4px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  background-color: #ebf0ee;
+  margin: 10px;
+}
+
+.languages + .languages {
+  margin-left: 25%;
+  margin-left: 3px;
+}
+
+.see-project {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-size: medium;
+  text-align: center;
+  color: white;
+  height: 60px;
+  width: 120px;
+  background: #36b37f;
+  border-radius: 4px;
+  border-style: none;
+}
+
+.container4 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item4 {
+  margin-bottom: 30px;
+}
+
+.container5 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item5 {
+  margin-bottom: 30px;
+}
+
+.container6 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item6 {
+  margin-bottom: 30px;
+}
+
+.container7 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item7 {
+  margin-bottom: 30px;
+}
+
+.container8 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item8 {
+  margin-bottom: 30px;
+}
+
+.container9 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border-style: none;
+  border-radius: 8px;
+}
+
```

Comment:
- [ ] The DRY principle states that there should be only one copy of important information. Cascading style sheets control the appearance of HTML documents. You are repeating yourself here by adding the same code that should be grouped together. For example

 display: flex;
  width: 290px;
  flex-direction: column;
  align-items: center;
  margin-left: auto;
  text-align: center;
  background: white;

Read [this](https://vanseodesign.com/css/dry-principles/) for more details